### ISSUE TITLE
Drag Drop External Events Not working

### ISSUE THREAD

**Issue by SrPatinhas** [CroudTech/vue-fullcalendar#117]
I'm trying to have external events to add to the calendar but the calendar doesn't recognize the drag over of the event.
It's almost the same as this one https://github.com/CroudTech/vue-fullcalendar/issues/72#issuecomment-387536660 but only with normal vue

**Comment by BrockReece** [comment]
Is it possible that you have 2 versions of jQuery running?

**Comment by SrPatinhas** [comment]
I dont think so, I removed the versions from laravel mix bootstrap.js and have only the one included in the header of the page

**Comment by BrockReece** [comment]
Are there any error messages?

**Comment by SrPatinhas** [comment]
no, It's all clean, its like it doesn't recognize the calendar

**Comment by BrockReece** [comment]
What happens if you try something like this in the console?

```js
$('#calendar').fullCalendar();
```

**Comment by SrPatinhas** [comment]
Nothing, it just resets the calendar as new

**Comment by BrockReece** [comment]
Hmm... are you able to add replicate the issue in a codesandbox or share some code?

**Comment by SrPatinhas** [comment]
sure, I don't have the code now, but later I can share the url for the page that I'm having the problem

**Comment by BrockReece** [comment]
Ok, thanks. I'm finishing up for the day now, but I will have a look in the morning.

**Comment by SrPatinhas** [comment]
ok, Thanks ;)

**Comment by SrPatinhas** [comment]
Ok, I tested now with the updated code and the `$('#calendar').fullCalendar();` doesn't work, it returns an error saying `Uncaught TypeError: $(...).fullCalendar is not a function` you can check the code in here https://v2.foodiebookings.com/calendar

**Comment by BrockReece** [comment]
Ah ok, so I am pretty sure this means that two instances of jQuery is running. Fullcalendar loads jQuery automatically if it isn't already loaded in the bundle. I have noticed you are loading jQuery in your head as opposed to loading it through webpack/mix etc.

https://fullcalendar.io/docs/installation

**Comment by BrockReece** [comment]
Removing those script tags and using something like this should work. (As did in the nuxt issue you linked to)

```js
import jquery from 'jquery'
import 'jquery-ui-bundle'

window.$ = window.jQuery = jquery
```
